src/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => {
+  return (
+    <nav
+      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      data-cy="Nav"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames(
+              'navbar-item',
+              { 'is-active': isActive },
+            )}
+          >
+            Home
+
+          </NavLink>
```

Comment:
It is better to move this code to a new component. And "Navigation" it's not the best name for this component, "NavBar" sounds better